### PR TITLE
Add pre commit hook for automatic code fix

### PR DESCRIPTION
**Pull request by maximest-pierre**
This PR adds a pre-commit hooks that automate the linting process.

### CODE REVIEW

**Comment by kauffj** [comment]
@maximest-pierre when this is ready, please separate the commit to simply add the hook from the commit that runs the hook and introduces a massive amount of formatting changes.

**Comment by maximest-pierre** [comment]
@kauffj Its ready to be look at

**Comment by kauffj** [comment]
Would like @lyoshenka to review this one instead of me.

File: dev.sh [hunk 1]
```diff
@@ -6,13 +6,16 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
...
+ cp "$DIR/data/config.php.example" "$DIR/data/config.php"
 fi
 
 if ! which $PHPBIN 2>/dev/null; then
-    PHPBIN=php
+   PHPBIN=php
 fi
 
+# Installing git hook
+./hooks/install.sh
```
Comment (lyoshenka) [inline dev.sh]: please use $DIR instead of `.`, since the current directory may not be what you think

File: docker.sh [hunk 2]
```diff
@@ -4,14 +4,18 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 
 if [ ! -e "$DIR/data/config.php" ]; then
-  cp "$DIR/data/config.php.example" "$DIR/data/config.php"
+ cp "$DIR/data/config.php.example" "$DIR/data/config.php"
 fi
 
+# Installing git hook
+./hooks/install.sh
```
Comment (lyoshenka) [inline docker.sh]: same here

**Comment by maximest-pierre** [comment]
Testing the pre-commit hook for the last two days and everything seems fine.

**Pull request merged**